### ISSUE TITLE
No active keyboard focus is present on Left pane of Workbooks window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#262]
### Steps to Reproduce
1.Launch Xamarin Workbook application and select platform and create a new workbook.
2. Navigate to Left pane (Project Tree) and try to navigate through the tree.

### Expected Behavior
Active keyboard should be present on Left pane and user should be able to traverse  these tree/table items using keyboard.

### Actual Behavior
No active keyboard focus is present on Left pane of Workbooks window.
User is unable to interact with these tree items  using arrow /tab keys etc. 
Note1: User is unable to invoke the Context menu for these left pane tree items.
Eg. In the left pane "NuGet Package" tree item having "Add Package.."  in context menu, but keyboard user is unable throw the same using keyboard. 
Note2: User is unable to expand collapse the tree items of the left pane.

### 
![kb7_leftpane_notinteractive](https://user-images.githubusercontent.com/22956130/39855293-fb8dc200-5448-11e8-8777-6696876f1305.png)
User Impact
User is unable to know, whether the left pane is getting keyboard the focus or not.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS2.4.7
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1815
